analyzers/tests/SonarAnalyzer.UnitTest/Rules/DeadStoresTest.cs

```diff
@@ -39,6 +40,17 @@ public class DeadStoresTest
                                     ParseOptionsHelper.FromCSharp8);
 #endif
 
+        [TestMethod]
```

Comment:
I would take it to the top, since the next one is also from Roslyn. But you can ignore this.

Reply:
Makes sense. I will not merge the `#if` statements even when I could. I'll get rid of the `NETStandardV2_1_0` issue and related `#if` in another PR.